<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot3dAssistantBase</class>
 <widget class="QWidget" name="Plot3dAssistantBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>263</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Expression:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="expression"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>1st Variable</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="variable1"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Minimum:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KLineEdit" name="min1"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Maximum:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KLineEdit" name="max1"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>2nd Variable</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="variable2"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Minimum:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KLineEdit" name="min2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Maximum:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KLineEdit" name="max2"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>expression</tabstop>
  <tabstop>variable1</tabstop>
  <tabstop>min1</tabstop>
  <tabstop>max1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
